packages/loader/driver-utils/src/runWithRetry.ts

```diff
@@ -61,13 +64,23 @@ export async function runWithRetry<T>(
                 }, err);
                 throw err;
             }
+
+            if (progress.cancel?.aborted){
+                throw new NonRetryableError(
+                    "runWithRetryAborted",
+                    undefined,
+                    DriverErrorType.genericError,
+                    { driverVersion: pkgVersion },
```

Comment:
Useful to stick fetchCallName as a property on error object to know what call was actually cancelled.
in the past we did this by changing name of event, i.e. something like `runWirhRetryAborted_${fetchCallName}`.
I'd prefer that way, as it's easier to work with data in this format (then have separate property, though we can do both).

Reply:
ok, though I'm not sure how to change the eventName here, since I'm throwing an error. Can I add eventName as an error property?